src/main/data_structure/string_exercise.py

```diff
@@ -0,0 +1,71 @@
+from src.main.logger.py_logger import PyLogger
+
+logger = PyLogger.get_configured_logger()
+
+
+# Count the number of characters (character appearance) in a string
+def char_appearance(string):
```

Comment:
change the function name. function name always should be verb

Reply:
ok